inc/admin.php

```diff
@@ -112,8 +113,8 @@ public function inline_bootstrap_script() {
 						s.src = "%s/v2/latest/optimole_lib" + v  + "%s.js"; 
 						b.appendChild(s);
 						w.optimoleData = {
-							lazyloadOnly: "optimole-lazy-only",
 							backgroundReplaceClasses: [%s],
+							lazyloadOnly: %s,
```

Comment:
I think it would be better to leave this as it was since there might be instances where this is used and instead use a flag for scale eg. `scalingEnabled` that is `bool`